### ISSUE TITLE
aws_workspaces_workspace / [compute_type_name] is missing support for "Graphics.g4dn | GraphicsPro.g4dn"

### ISSUE THREAD

**Issue by darkb1u3** [hashicorp/terraform-provider-aws#25112]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Resource "aws_workspaces_workspace" / [compute_type_name] is missing support for "Graphics.g4dn | GraphicsPro.g4dn" types.

Bundles:
https://aws.amazon.com/workspaces/features/#Amazon_WorkSpaces_Bundles

Was recently added:
https://aws.amazon.com/about-aws/whats-new/2022/04/amazon-workspaces-graphics-g4dn-bundles/

### New or Affected Resource(s)

aws_workspaces_workspace / [compute_type_name]

### Potential Terraform Configuration

```hcl
variable "instance_g4dn" {
    default = "GRAPHICS_G4DN"
}

workspace_properties {
    compute_type_name                         = "${var.instance_g4dn}"
```

│ Error: expected workspace_properties.0.compute_type_name to be one of [VALUE STANDARD PERFORMANCE POWER POWERPRO GRAPHICS GRAPHICSPRO], got GRAPHICS_G4DN

### References

* #0000

**Comment by justinretzolk** [comment]
Hey @darkb1u3 👋 Thank you for taking the time to raise this! The `aws_workspaces_workspace` resource [uses `workspaces.Compute_Values()`](https://github.com/hashicorp/terraform-provider-aws/blob/e70faae038841b83dc077de02ab7a07bb7c6726e/internal/service/workspaces/workspace.go#L82) to validate the value provided in the `compute_type_name` argument. This function was [updated in AWS Go SDK version `v1.43.37`](https://github.com/aws/aws-sdk-go/pull/4358), which was [pulled into the AWS provider in version `4.10.0`](https://github.com/hashicorp/terraform-provider-aws/pull/24193). Initially, I expected that this would mean that version `4.10.0` would correct the issue, however, the move to using `workspaces.Compute_Values()` didn't actually happen until [AWS Provider version `4.11.0`](https://github.com/hashicorp/terraform-provider-aws/pull/24286).

In short, upgrading to AWS Provider version `4.11.0` will resolve this error 🙂
